### PR TITLE
Responsive navbar

### LINKED ISSUE #77: Responsive Navbar
image

- Create a responsive navbar (look into Bootstrap Navbar)
- Top navbar will be for laptop screens, button navbar will be for mobile screens
- Use `<Link/>` component to link to various pages

### PR DESCRIPTION
**Pull request by skyguy-netizen**

### CODE REVIEW

**Comment by skyguy-netizen** [comment]
file:///home/aarav/Pictures/Screenshots/Screenshot%20from%202022-08-22%2020-02-39.png
file:///home/aarav/Pictures/Screenshots/Screenshot%20from%202022-08-22%2020-03-03.png
file:///home/aarav/Pictures/Screenshots/Screenshot%20from%202022-08-22%2020-03-11.png

File: components/Navigation.js [hunk 1]
```diff
@@ -1,7 +1,74 @@
...
 
 const Navigation = () => {
-	return <div>Navigation</div>;
+	return (
+		<Navbar collapseOnSelect bg='light' expand='lg'>
+			<Navbar.Brand>
+				<Link href='/' passHref>
+					<Nav.Link eventKey='1'>
+						<img
+							src='acmlogo.png'
```
Comment (shahdivyank) [inline components/Navigation.js]: can you change the image name to `acm-ucr-logo.png` Valerie beat you to the logo naming rights https://github.com/acm-ucr/acm-dev/pull/108

File: components/Navigation.js [hunk 2]
```diff
@@ -1,7 +1,74 @@
...
+							width='40%'
+							height='40%'
+						/>
+					</Nav.Link>
+				</Link>
+			</Navbar.Brand>
+			<Navbar.Toggle aria-controls='basic-navbar-nav' />
+			<Navbar.Collapse id='basic-navbar-nav'>
+				<Nav className='ml-auto'>
+					<ul className='navbar-nav ml-auto float-rightmb-2 mb-lg-0'>
```
Comment (shahdivyank) [inline components/Navigation.js]: add a space

File: components/Navigation.js [hunk 3]
```diff
@@ -1,7 +1,74 @@
...
+	return (
+		<Navbar collapseOnSelect bg='light' expand='lg'>
+			<Navbar.Brand>
+				<Link href='/' passHref>
+					<Nav.Link eventKey='1'>
+						<img
+							src='acmlogo.png'
+							alt=''
+							width='40%'
+							height='40%'
```
Comment (shahdivyank) [inline components/Navigation.js]: Use Tailwind

File: components/Navigation.js [hunk 4]
```diff
@@ -1,7 +1,74 @@
...
 const Navigation = () => {
-	return <div>Navigation</div>;
+	return (
+		<Navbar collapseOnSelect bg='light' expand='lg'>
+			<Navbar.Brand>
+				<Link href='/' passHref>
+					<Nav.Link eventKey='1'>
+						<img
+							src='acmlogo.png'
+							alt=''
```
Comment (shahdivyank) [inline components/Navigation.js]: Add alt text

File: components/Navigation.js [hunk 5]
```diff
@@ -1,7 +1,74 @@
...
+							<Link align='right' href='/about' passHref>
+								<Nav.Link
+									className='text-black no-underline'
+									eventKey='2'
+								>
+									about
+								</Nav.Link>
+							</Link>
+						</li>
+						<li className='nav-item active nav-link font-lexend font-color-black'>
```
Comment (shahdivyank) [inline components/Navigation.js]: Apply `font-lexend` at the top level `<Navbar/>` and remove from children

File: components/Navigation.js (hunk 2 above)
Comment (shahdivyank) [inline components/Navigation.js]: remove `ui` and `li`, just use the `Link` components

File: components/Navigation.js [hunk 6]
```diff
@@ -1,7 +1,74 @@
 import React from "react";
+import { Nav, Navbar } from "react-bootstrap";
+import Link from "next/link";
 
 const Navigation = () => {
-	return <div>Navigation</div>;
+	return (
+		<Navbar collapseOnSelect bg='light' expand='lg'>
```
Comment (shahdivyank) [inline components/Navigation.js]: make the font-size larger at the top level

Comment (shahdivyank) [inline components/Navigation.js]: remove `bg-light` that makes it gray

**Review by shahdivyank — CHANGES_REQUESTED** [review]

File: components/Navigation.js [hunk 7]
```diff
@@ -1,7 +1,48 @@
...
+							src='acm-ucr-logo.png'
+							className='w-2/5 h-2/5'
+							alt='acm@ucr logo'
+						/>
+					</Nav.Link>
+				</Link>
+			</Navbar.Brand>
+			<Navbar.Toggle aria-controls='basic-navbar-nav' />
+			<Navbar.Collapse id='basic-navbar-nav'>
+				<Nav className='ml-auto nav-item active nav-link float-right mb-2 mb-lg-0 no-underline'>
```
Comment (shahdivyank) [inline components/Navigation.js]: dont use float, use flex and justify-between

File: components/Navigation.js [hunk 8]
```diff
@@ -1,7 +1,48 @@
...
+							board
+						</Nav.Link>
+					</Link>
+					<Link href='/events' passHref>
+						<Nav.Link className='text-black' eventKey='4'>
+							events
+						</Nav.Link>
+					</Link>
+					<Link href='/join' passHref>
+						<Nav.Link className='text-black' eventKey='5'>
```
Comment (shahdivyank) [inline components/Navigation.js]: use `text-acm-black`

**Review by shahdivyank — CHANGES_REQUESTED** [review]

**Comment by skyguy-netizen** [comment]
Screenshot from 2022-08-23 20-12-14
Screenshot from 2022-08-23 20-12-07
Screenshot from 2022-08-23 20-11-42

**Comment by shahdivyank** [comment]
Have it collapse on small screens, instead of large

**Comment by skyguy-netizen** [comment]
Screenshot from 2022-08-24 19-13-18
Screenshot from 2022-08-24 19-13-13

**Review by shahdivyank — APPROVED** [review]

**Review by Sarah-Rami27 — APPROVED** [review]

**Pull request merged**